pkg/descheduler/strategies/nodeutilization/lownodeutilization_test.go

```diff
@@ -552,70 +562,54 @@ func TestLowNodeUtilization(t *testing.T) {
 			expectedPodsEvicted: 3,
 		},
 		{
-			name: "without priorities, but only other node doesn't match pod node affinity for p4 and p5",
+			name:                   "without priorities, but only other node doesn't match pod node affinity for p4 and p5",
+			useDeviationThresholds: false,
+			thresholds:             api.ResourceThresholds{v1.ResourceCPU: 30, v1.ResourcePods: 30},
```

Comment:
Would you mind putting back the original formatting? I.e. `api.ResourceThresholds{v1.ResourceCPU: 30, v1.ResourcePods: 30},` into:
```
api.ResourceThresholds{
	v1.ResourceCPU: 30,
	v1.ResourcePods: 30,
},
```
The same for other occurrences.